tests/lib/rules/comma-dangle.js

```diff
@@ -11,22 +11,22 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-var eslint = require("../../../lib/eslint"),
-    ESLintTester = require("../../../lib/testers/eslint-tester");
+var rule = require("../../../lib/rules/comma-dangle"),
+    RuleTester = require("../../../lib/testers/rule-tester");
 
 //------------------------------------------------------------------------------
 // Tests
 //------------------------------------------------------------------------------
 
-var eslintTester = new ESLintTester(eslint);
+var ruleTester = new RuleTester();
 
-eslintTester.addRuleTest("lib/rules/comma-dangle", {
+ruleTester.run("comma-dangle", rule, {
     valid: [
         "var foo = { bar: 'baz' }",
         "var foo = {\nbar: 'baz'\n}",
         "var foo = [ 'baz' ]",
         "var foo = [\n'baz'\n]",
-        "[,,]",
+        "[,]",
         "[\n,\n,\n]",
         "[,]",
```

Comment:
With this change, lines 29 and 31 seem to be duplicates.  

Same goes for 48/50, and 64/66